earthpy/tests/test_mask.py

```diff
@@ -8,18 +8,14 @@
 @pytest.fixture
 def im():
     """Create image arr for masking"""
-    arr = np.array(
-        [[0, 1, 8, 7, 5], [3, 8, 8, 7, 0], [3, 0, 0, 9, 9], [5, 5, 7, 4, 5]]
-    )
+    arr = np.array([[0, 1], [2, 3]])
```

Comment:
heya @mbjoseph i don't love that this array is so simple that only one val for each potential mask element exists. I also dont love that both arrays have the same values because it just leaves room for confusion on many levels. . my thought was the mask array would have a subset of values in the larger array to be masked. can we make this a bit bigger and have larger values in the initial array.

Reply:
@lwasser thanks for the review! It's awesome to be able to sort out what makes for good tests together.

Is there functionality in `mask_pixels` related to the following, that is not captured in the current tests?

1. `im_mask` having a subset of values in `im`
2. Having larger arrays for `im` and `im_mask`
3. Having larger values in the `im` fixture

If so, then we should definitely test that functionality! If not, then I think there's value in having the tests be as simple as possible (so that they are easy to understand). 

If using the same array for both `im` and `im_mask` is confusing, then that's bad. I experimented with some other ideas, but it seemed easiest to go with this simple 2x2 array, because it's small and easy to understand. Do you have an alternative in mind that would be easier to grasp?

Reply:
sure. i worry that if both arrays have the same value, and there is only one instance of that value, that something could "work" by mistake. 

I'd prefer to see a slightly larger array that duplicated values to be masked so we see that it works on n+1. id also prefer that the array to do the masking has fewer values than the array to be masked. 

So maybe something like

```
arr = np.array([[0, 1, 2 ,3], [2, 3, 3, 2]])
mask_arr = np.array([[0, 1, 2, 2], [0,2, 2,1]])
```

i'd like to see an array where 
1. more than 1 value that is to be masked is masked so we know that works - that being it masks more than 1 value and
2. the mask array itself is not the same as the input array as you would expect it to be IRL.
 
In this cause i really want to see an example that represents what the data would actually be. if both masks were the same THEN you wouldn't need the second parameter at all, you'd just set all vals in the main array to NAN.

Reply:
Ok @lwasser  have a look at the updated fixtures, for which: 

- the fixtures are larger
- there are values in `im` that are not in `im_mask`
- there are duplicated values in `im_mask`

If we are still worried about things working by mistake, then we need test cases for those situations.

Reply:
cool. did you add a commit? i don't see it if you did.

Reply:
i see it now. github is weirdly slow today. shall we merge this not @mbjoseph  or did i somehow miss addressing other things you'd like to see? thank you!!